RadixWallet/Core/DesignSystem/Components/PoolUnitView.swift

```diff
@@ -16,24 +16,13 @@ public struct PoolUnitView: View {
 		Button(action: onTap) {
 			VStack(alignment: .leading, spacing: .zero) {
 				HStack(spacing: .zero) {
-					LoadableImage(url: viewState.poolIcon, size: .fixedSize(.verySmall)) {
-						ZStack {
-							Circle()
-								.fill(.app.gray4)
-								.frame(width: .large1, height: .large1)
-							Image(asset: AssetResource.poolUnits)
-								.resizable()
-								.frame(.verySmall)
-						}
-					}
-					.padding(.trailing, .medium3)
+					Thumbnail(.poolUnit, url: viewState.poolIcon, size: .extraSmall)
+						.padding(.trailing, .medium3)
 
 					VStack(alignment: .leading, spacing: 0) {
-						if let poolName = viewState.poolName {
-							Text(poolName)
-								.textStyle(.body1Header)
-								.foregroundColor(.app.gray1)
-						}
+						Text(viewState.poolName ?? L10n.TransactionReview.poolUnits)
```

Comment:
I am not entirely sure we should have the fallback name, but probably better than empty string.

Reply:
Zeplin has a it though